features/admin.administrators.v1/wizard/steps/admin-user-basic.tsx

```diff
@@ -308,170 +341,200 @@ export const AddAdminUserBasic: React.FunctionComponent<AddAdminUserBasicProps>
+                                                                    value, window["AppUtils"]
+                                                                        .getConfig().extensions.
+                                                                        collaboratorUsernameRegex
+                                                                )
+                                                            )
+                                                        ) {
+                                                            validation.isValid = false;
+                                                            validation.errorMessages.push(
+                                                                USERNAME_REGEX_VIOLATION_ERROR_MESSAGE
+                                                            );
+                                                        }
+                                                    } }
+                                                    type="email"
+                                                    component={ TextFieldAdapter }
+                                                    tabIndex={ 5 }
+                                                    maxLength={ 50 }
+                                                />
+                                            </Grid.Column>
+                                        </Grid.Row>
+                                        <Grid.Row columns={ 1 }>
+                                            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 12 }>
+                                                <FinalFormField
+                                                    data-componentid={ `${ componentId }-external-form-role-dropdown` }
+                                                    label={ "Roles" }
+                                                    name="roles"
+                                                    options={ userRoleOptions }
+                                                    requiredErrorMessage={ t(
+                                                        "user:forms.addUserForm.inputs.domain.validations.empty"
+                                                    ) }
+                                                    placeholder="Select roles"
+                                                    component={ AutocompleteFieldAdapter }
+                                                    required={ true }
+                                                    value={ userRoleOptions[0]?.value }
+                                                    tabIndex={ 1 }
+                                                    multipleValues
+                                                />
+                                                <Hint>
+                                                    { "Select a role to assign to the user." +
+                                    " The access level of the user is determined by the role." }
```

Comment:
Include `i8n`

Reply:
This was existing text in the invite privileged user modal and I have just moved it around. Anyways, will send a fix for this in a followup PR.